apps/coordinator/src/components/Wallet/TransactionPreview.jsx

```diff
@@ -186,6 +231,23 @@ class TransactionPreview extends React.Component {
             <h3>Total</h3>
             <div>{satoshisToBitcoins(BigNumber(inputsTotalSats || 0))} BTC</div>
           </Grid>
+          <Grid item xs={12}>
+            <h3>Waste Analysis</h3>
+            <Typography gutterBottom>
+              Spend Waste Amount: {wasteAmount.toFixed(2)} Sats
+            </Typography>
+            <Slider
+              value={longTermFeeEstimate}
+              min={1}
+              max={3000}
+              step={1}
+              onChange={this.handleLongTermFeeEstimateChange}
+              aria-labelledby="long-term-fee-estimate-slider"
+            />
+            <Typography id="long-term-fee-estimate-slider" gutterBottom>
+              Long Term Fee Estimate (L): {longTermFeeEstimate} sats/vB
```

Comment:
Can we add some info tooltips maybe to explain what the waste analysis, the SWA and the L are?

Reply:
Sure